src/posix.mak

```diff
@@ -208,7 +209,7 @@ SRC = win32.mak posix.mak osmodel.mak \
 ROOT_SRC = $(ROOT)/root.h \
 	$(ROOT)/array.h \
 	$(ROOT)/rmem.h $(ROOT)/rmem.c $(ROOT)/port.h $(ROOT)/port.c \
-	$(ROOT)/man.c \
```

Comment:
I hate to nitpick, but I hate the name `rmem_operators.c`. It's overlong, and has that embedded underscore. How about `newdelete.c` ? Everything else looks good.